core/src/Cabana_LinkedCellList.hpp

```diff
@@ -141,10 +141,13 @@ class LinkedCellList
                  grid_delta[2] )
         , _cell_stencil( grid_delta[0], 1.0, grid_min, grid_max )
         , _sorted( false )
+        , _binned( false )
     {
         std::size_t np = positions.size();
         allocate( totalBins(), np );
         build( positions, 0, np );
+        getParticleBins();
```

Comment:
Can we just call this in `build` instead (called in every constructor) and then not have to store/check if it's binned?

Reply:
We probably can. I'll try that We also need to call it in permute as well since we change the particle IDs there.

Reply:
Good point